k8s/README.md

```diff
@@ -25,7 +25,7 @@ k8s://[namespace/]fieldSelector/[selector] - k8s://default/fieldSelector/metadat
 
 ## 2. Create the Kubernetes service account in the namespace:
 
-`kubectl create serviceaccount goreplay --namespace goreplay`
+`kubectl -n goreplay -f serviceaccount.yaml apply`
```

Comment:
Please do not change anything inside goreplay repo except files inside middleware/traffic_enrich. 
We, I mean probably me only, do not want to maintain divergences from the original repo.  We only want to add a new middleware.
So, move or merge these instructions to the system repo readme.